modules/default/calendar/calendar.js

```diff
@@ -396,7 +396,7 @@ Module.register("calendar", {
 					var descCell = document.createElement("td");
 					descCell.className = "location";
 					descCell.colSpan = "2";
-					descCell.innerHTML = event.location;
+					descCell.innerHTML = this.titleTransform(event.location);
```

Comment:
maybe rename the function to show that it not only transform the title but now it transforms other stuff too?

Reply:
Yes I also thought about that. I wanted to create this PR and ask the editors and maintainers here, what they might prefer better (just rename the function to better represent what it does or do this completely independent from the title and have a seperate variable to control the truncating of the location title).

Reply:
I couldn't come up with a better name for this function (since the event location title is also a title), but I think truncate better represent what this function does right now.